src/propertySelector/propertySelector.js

```diff
@@ -0,0 +1,150 @@
+import component from 'ui/component';
```

Comment:
**chore:** The license header is missing.

Reply:
Thanks, fixed